sdk/include/c++/stlport/stl/config/_windows.h

```diff
@@ -205,7 +205,7 @@ _STLP_IMPORT_DECLSPEC void _STLP_STDCALL OutputDebugStringA(const char* lpOutput
  * to avoid macro definition conflict. */
 #  if !defined (_WIN64)
 /* Under 32 bits platform we rely on a simple InterlockedExchange call. */
-#    if defined (__cplusplus) && defined(__BUILDING_STLPORT)
```

Comment:
I had to hack this, because STLPInterlockedExchangePointer is used in _threads.h, which is included through stl headers by drwtsn32.

Reply:
Just comment out that second defined(...) instead of removing it? (this might help remembering what the original line was.)